scripts/bulk-mailer/index.js

```diff
@@ -0,0 +1,111 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const chunk = require('lodash.chunk')
+const config = require('../../config').getProperties()
+const error = require('../../lib/error')
+const P = require('../../lib/promise')
+const readUserRecords = require('./read-user-records')
+const sendEmailBatches = require('./send-email-batches')
+const Senders = require('../../lib/senders')
+const Translator = require('../../lib/senders/translator')
+const UserRecordNormalizer = require('./normalize-user-records')
+const WriteToStreamSenderMock = require('./nodemailer-mocks/stream-output-mock')
+const WriteToDiskSenderMock = require('./nodemailer-mocks/write-to-disk-mock')
+
+
+const bouncesMock = {
+  check: () => P.resolve()
+}
+
+const logMock = {
+  amplitudeEvent: () => {},
+  error: console.error,
+  info: () => {},
+  trace: () => {}
```

Comment:
Log all the things!

Right now, when I run `./scripts/bulk-mailer.js` the only message is a `console.log('done')`.

This script should log each intent (`sending foo@example`), and result (`sent to foo@example.com at ${time} with SMTP Message-Id ${messageId}).

If there is some configuration setting I need to make, please let me know.

Reply:
(Actually, I don't know what I'd do with the message-id, but it makes me warm and fuzzy that the providers response actually had assigned one).